packages/cozy-stack-client/package.json

```diff
@@ -12,9 +12,11 @@
   },
   "dependencies": {
     "detect-node": "2.0.4",
-    "mime-types": "2.1.20",
     "qs": "6.6.0"
   },
+  "optionalDependencies": {
+    "mime-types": "2.1.20"
+  },
```

Comment:
As I understand, if we put mime-types in optionalDependencies, yarn&npm will try by default to install it. I don't know if we shouldn't put in `peerDependencies`.

Reply:
peerDependency allows us to define the version of the dependency outside of the lib. It's aimed a not duplicating modules across different libraries. peerDependencies are required dependencies, they will always be present in the final lib